tools/common/Driver.cs

```diff
@@ -202,6 +261,56 @@ static bool ParseOptions (Application app, Mono.Options.OptionSet options, strin
 					app.EnvironmentVariables.Add (name, value);
 				}
 			);
+			options.Add ("registrar:", "Specify the registrar to use (dynamic, static or default (dynamic in the simulator, static on device))", v => {
+				var split = v.Split ('=');
+				var name = split [0];
+				var value = split.Length > 1 ? split [1] : string.Empty;
+
+				switch (name) {
+				case "static":
+					app.Registrar = RegistrarMode.Static;
+					break;
+				case "dynamic":
+					app.Registrar = RegistrarMode.Dynamic;
+					break;
+				case "default":
+					app.Registrar = RegistrarMode.Default;
+					break;
+#if MMP
+				case "partial":
+				case "partial-static":
+					app.Registrar = RegistrarMode.PartialStatic;
+					break;
+				case "il":
+					app.Registrar = RegistrarMode.Dynamic;
```

Comment:
Is `il` really an exact alias for dynamic? Does it need to be deprecated?

Reply:
building on that comment, we can move the case next to the dynamic one.

Reply:
I just removed the `il` option.

Reply:
👍